test/quicer_snb_SUITE.erl

```diff
@@ -230,6 +231,85 @@ tc_slow_conn(Config) ->
                end),
   ok.
 
+tc_stream_owner_down(Config) ->
```

Comment:
Ease to combine and check with three players in one test, I fall in love with `snk`, thank you! @k32